contrib/native/client/readme.macos

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+MacOS build (tested on OS X El Capitan)
+
+Install Prerequisites
+---------------------
+
+0.1) Install XCode 
+  Download an istall from here: https://developer.apple.com/xcode/downloads/
+  In Terminal, install the command line tools 
+    $> xcode-select --install
+
+0.2) Install brew following the instructions here: http://brew.sh/
+
+1) CMAKE 2.8 or above
+  Download and install Cmake : https://cmake.org/download/
+
+2.1) Install protobuf
+  $> brew install protobuf
+
+2.2) Install zookeeper
+  $> brew install zookeeper
+
+2.3) Install boost
+  $> brew install boost
+
+2.3.1) For production builds, see the readme.boost file  
+  
+  
+(Optional) Refresh protobuf source files
+----------------------------------------
+When changes have been introduced to the protocol module, you might need to refresh the protobuf C++ source files too.
+  $> cd DRILL_DIR/contrib/native/client
+  $> mkdir build
+  $> cd build && cmake3 -G "XCode" -D CMAKE_BUILD_TYPE=Debug ..
+  $> xcodebuild -project drillclient.xcodeproj -configuration ${BUILDTYPE} -target fixProtobufs
+  $> xcodebuild -project drillclient.xcodeproj -configuration ${BUILDTYPE} -target cpProtobufs
+
+Open a pull request with the changes to DRILL_DIR/contrib/native/client/src/protobuf
+
+Build drill client
+-------------------
+  $> cd DRILL_DIR/contrib/native/client
+  $> mkdir build
+  $> cd build && cmake3 -G "XCode" -D CMAKE_BUILD_TYPE=Debug ..
+  $> xcodebuild -project drillclient.xcodeproj -configuration ${BUILDTYPE} -target ALL_BUILD
+
+
+XCode IDE
+---------
+  You can open the drillclient.xcodeproj file in the XCode ide and run/debug as with any other command line app
+
+Test
+----
+Run query submitter from the command line
+  $> querySubmitter query='select * from dfs.`/Users/pchandra/work/data/tpc-h/customer.parquet`' type=sql connectStr=local=10.250.0.146:31010 api=async logLevel=trace user=yourUserName password=yourPassWord
+
+Valgrind
+--------
+  If you can get valgrind to build and run on macos, please update the instuctions here
```

Comment:
Ah, could never get brew to install it for me. I had tried to build valgrind way back and its uninstall did not clean up leaving me with a system on which the brew install doesn't work either.
Also, valgrind is not updated for the latest MacOs and there is no valkyrie. :( 
Can you double check the instructions I wrote down? I just copied the stuff from the linux instructions.